core/modules/utils/dom/dom.js

```diff
@@ -66,13 +66,23 @@ exports.toggleClass = function(el,className,status) {
 
 /*
 Get the first parent element that has scrollbars or use the body as fallback.
+From https://stackoverflow.com/questions/35939886/find-first-scrollable-parent
 */
-exports.getScrollContainer = function(el) {
+exports.getScrollContainer = function(el,includeHidden) {
 	var doc = el.ownerDocument;
-	while(el.parentNode) {
-		el = el.parentNode;
-		if(el.scrollTop) {
-			return el;
+	var style = getComputedStyle(el);
+	var excludeStaticParent = style.position === "absolute";
+	var overflowRegex = includeHidden ? /(auto|scroll|hidden)/ : /(auto|scroll)/;
+	if(style.position === "fixed") {
+		return doc.body;
+	}
+	for(var parent=el;(parent = parent.parentElement);) {
```

Comment:
For better readability IMO the line should look like this. No extra braces:
`for(var parent=el; parent=parent.parentElement;) {`

Reply:
thanks @pmario, I'm about to fix this

Reply:
Please also have a look at the whitespace. There is a space after `;` but no space before and after `=` 
See core for loops: https://github.com/Jermolene/TiddlyWiki5/blob/124b49456a4dc2b75ecab17e9be5180464e4d9b1/boot/boot.js#L83  .. They should look like this